cdk/infrastructure/constructs/backend.py

```diff
@@ -119,6 +132,13 @@ def __init__(
         self._run_batch_upgrade_automatically()
         self._run_update_mapping_automatically()
         self._add_alarms()
+        self._add_dashboard()
+
+    def _define_log_driver_for_pyramid_container(self) -> None:
```

Comment:
Can you put this right above where application container gets defined?

Reply:
`_define_log_driver_for_pyramid_container`
Can we call it `_define_log_driver_for_application_container` to be consistent?